viper.go

```diff
@@ -1811,10 +1811,16 @@ func (v *Viper) getConfigFile() (string, error) {
 func (v *Viper) searchInPath(in string) (filename string) {
 	jww.DEBUG.Println("Searching for config in ", in)
 	for _, ext := range SupportedExts {
-		jww.DEBUG.Println("Checking for", filepath.Join(in, v.configName+"."+ext))
-		if b, _ := exists(v.fs, filepath.Join(in, v.configName+"."+ext)); b {
-			jww.DEBUG.Println("Found: ", filepath.Join(in, v.configName+"."+ext))
-			return filepath.Join(in, v.configName+"."+ext)
+		configFilePlusExt := ""
```

Comment:
This is a personal preference, but nil value variable declarations make more sense as `var name type`. E.g. `var configPlusExt string`

Reply:
Done. Thanks.